cpp/log/cert_checker_test.cc

```diff
@@ -336,7 +336,9 @@ TEST_F(CertCheckerTest, AcceptNoBasicConstraintsAndMd2) {
   Cert ca(ca_pem);
   // Verify testdata properties: CA is legacy root.
   ASSERT_EQ("md2WithRSAEncryption", ca.PrintSignatureAlgorithm());
-  ASSERT_EQ(Cert::FALSE, ca.HasBasicConstraintCATrue());
+  StatusOr<bool> has_ca_constraint = ca.HasBasicConstraintCATrue();
+  ASSERT_TRUE(has_ca_constraint.ok() &&
```

Comment:
More of the same as my `cpp/log/cert_test.cc` comment here (and the other bit below).